frontend/src/screens/observing-conditions/observing-conditions.js

```diff
@@ -109,7 +113,7 @@ class TrendAnalysis extends React.Component {
 
   handleChangeDatePeriod = evt => {
     this.setState({ datePeriod: evt.target.value, endDate: moment().format() });
-    this.loadStart();
+    this.loadEnd();
```

Comment:
If you press submit more than once loading is appearing forever.

Not sure if this line here is needed but this function must be reallocated to fix this issue.